### PR TITLE
Vision snippets.

### PR DESCRIPTION
**Pull request by jmdobry**

### CODE REVIEW

File: vision/detect.js [hunk 1]
```diff
@@ -0,0 +1,457 @@
...
+
+const Vision = require('@google-cloud/vision');
+const Storage = require('@google-cloud/storage');
+
+// [START vision_face_detection]
+function detectFaces (fileName) {
+  // Instantiates a client
+  const vision = Vision();
+
+  // The local image file on which to perform face detection
```
Comment (gguuss) [inline vision/detect.js]: Any reason to leave this?

File: vision/detect.js [hunk 2]
```diff
@@ -0,0 +1,457 @@
...
+  // fileName = '/path/to/image.jpg';
+
+  // Performs face detection on the local file
+  return vision.detectFaces(fileName)
+    .then((results) => {
+      const faces = results[0];
```
Comment (gguuss) [inline vision/detect.js]: Nit. Prefer [four spaces on breaks when wrapping preceding line](https://google.github.io/styleguide/javascriptguide.xml?showone=Code_formatting#Code_formatting):

```
  return vision.detectFaces(fileName)
      .then((results) => { 
        const faces = results[0];
```

That said, your aforementioned two-space style persists throughout and is consistent with anonymous function spacing so you're free to ignore as this style is consistent elsewhere and would require more changes to remain so, which is more important.

File: vision/system-test/detect.test.js [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+const bucketName = `nodejs-docs-samples-test-${uuid.v4()}`;
+const cmd = `node detect.js`;
+const cwd = path.join(__dirname, `..`);
+const files = [
+  `face_no_surprise.jpg`,
+  `landmark.jpg`,
+  `logos.png`,
+  `text.jpg`,
+  `wakeupcat.jpg`
```
Comment (gguuss) [inline vision/system-test/detect.test.js]: Are you using `water.jpg` for any tests?

File: vision/detect.js (hunk 1 above)
Comment (jmdobry) [inline vision/detect.js]: Removed.

File: appengine/system-test/all.test.js [hunk 4]
```diff
@@ -129,13 +129,13 @@ var sampleTests = [
...
     msg: 'Logged'
   },
-  {
-    dir: 'appengine/loopback',
-    cmd: 'node',
-    args: ['server/server.js'],
-    msg: 'started',
-    code: 304
-  },
+  // {
```
Comment (jmdobry) [inline appengine/system-test/all.test.js]: Added a comment.

File: vision/system-test/detect.test.js (hunk 3 above)
Comment (jmdobry) [inline vision/system-test/detect.test.js]: One of the other samples refers to it.

File: vision/detect.js (hunk 2 above)
Comment (jmdobry) [inline vision/detect.js]: We'll stick with 2 spaces.

**Pull request merged**